build/pause/Makefile

```diff
@@ -12,42 +12,71 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY:	build push
+.PHONY: all push push-legacy-too container clean
 
-TAG=2.0
-REGISTRY?="gcr.io/google_containers"
+REGISTRY ?= gcr.io/google_containers
+IMAGE = $(REGISTRY)/pause-$(ARCH)
+LEGACY_AMD64_IMAGE = $(REGISTRY)/pause
+
+TAG = 3.0
 
 # Architectures supported: amd64, arm, arm64 and ppc64le
-ARCH?=amd64
-GOLANG_VERSION=1.6
-TEMP_DIR:=$(shell mktemp -d)
+ARCH ?= amd64
```

Comment:
can you add a var that is a list of supported ARCH values since you have to un-edit CHANGELOG :)